samples/QmlArExample/qml/main.qml

```diff
@@ -72,33 +84,55 @@ ApplicationWindow {
                 arcGISArView.planeColor = "";
         }
 
-        onEmptySceneClicked: changeScene("qrc:/qml/scenes/EmptyScene.qml");
-        onStreetsSceneClicked: changeScene("qrc:/qml/scenes/StreetsScene.qml");
-        onImagerySceneClicked: changeScene("qrc:/qml/scenes/ImageryScene.qml");
+        onEmptySceneClicked: changeScene("qrc:/qml/scenes/EmptyScene.qml", 0.000001);
+        onStreetsSceneClicked: changeScene("qrc:/qml/scenes/StreetsScene.qml", 0.000001);
+        onImagerySceneClicked: changeScene("qrc:/qml/scenes/ImageryScene.qml", 0.000001);
```

Comment:
Should these hard-coded values be stored somewhere with names? These aren't helpful like this to users.

Reply:
These values mean nothing in particular, this depend on the scene and translation factor, camera, etc. I can create names but this will be `EmptySceneFactor`, `StreetsSceneFactor`, etc. I don't know if it's useful.

Reply:
@JamesMBallard Do you want I change that?

Reply:
@GuillaumeBelz yes please make that change. Even if these values are specific to these datasets, it will help users if they have some meaningful name, like `readonly property double emptySceneFactor: 0.000001`